packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -297,14 +379,22 @@ export const AddNewTrustline: FC = () => {
           </>
         }
         transaction={TransactionStatus.Rejected}
+        {...(params.inAppCall ? { onClick: () => navigate(HOME_PATH) } : {})}
       />
     );
   }
 
   if (errorDifference) {
```

Comment:
```suggestion
  if (errorDifference && !params.inAppCall) {
```

Reply:
We still want to render what's below even if it's an in app call